infrastructure/tooling/templates/k8s/ingress.yaml

```diff
@@ -6,14 +6,24 @@ metadata:
   annotations:
     'kubernetes.io/ingress.global-static-ip-name': '{{ .Values.ingressStaticIpName }}'
     'ingress.gcp.kubernetes.io/pre-shared-cert': '{{ .Values.ingressCertName }}'
+    'cert-manager.io/cluster-issuer': '{{ .Values.certManagerClusterIssuerName | default "" }}'
```

Comment:
those extra annotations will do no harm when left blank to any existing deployment.

Plus mozilla is overriding those for moz-specific deployments